command/certificate/fingerprint.go

```diff
@@ -80,6 +80,18 @@ debugging invalid certificates remotely.`,
 			},
 			flags.ServerName,
 			command.FingerprintFormatFlag("hex"),
+			cli.StringFlag{
+				Name:  "alg",
+				Value: "SHA-256",
+				Usage: `The <alg> to hash the certificate. 
+ If unset, default is SHA-256. 
+  
+: <alg> is a case-sensitive string and must be one of: 
+  
+     **SHA-256**
+
+     **SHA-1** `,
+			},
```

Comment:
In `step crypto hash digest` we use `sha1` and `sha256`, we should be consitent

Reply:
> Hi @1984gobrr,
> 
> Thank you for your contribution!
> 
> We've discussed adding this functionality to the CLI with the team and concluded we'd like to gate usage of `SHA1` behind the `--insecure` flag, as `SHA1` is cryptographically broken and people generally shouldn't be relying on it. Using `--insecure` still allows one to use the CLI for legacy interoperability cases, but is also a small nudge in the direction of using more secure hashing algorithms. Could you add usage of `--insecure` to your PR?
> 
> One other thing I noted is that there are a couple more functionalities in the CLI that also allow providing/switching hash algorithms. The way this PR makes it possible to set the `SHA1` algorithm to use is slightly different than how it's done in the other commands. At this time I don't think we'll need to support a complete spectrum of hash algorithms, so I would propose to change the `--alg=<hash>` flag to a `--sha1` bool flag, similar to what's done in `command/crypto/key/fingerprint.go`. Providing `--sha1` would then override the default SHA-256 algorithm. The additional benefit of this is that users don't have to get the algorithm name right nor do they have to use the right case (lower vs. upper). I realize this is slightly different from what was suggested to be done in the issue (e.g. this way of doing it exists in `step crypto hash`), but I think it's sensible to do for this use case and very similar to what `step crypto key fingerprint` does. Any potential future changes to this may result in doing the hash selection slightly differently, but in that case we'd probably want to make it a bit more reusable and also make it more user friendly in terms of casing and algorithm naming.

Ok, I will do that today :smile:

Reply:
It is now possible to do `step certificate fingerprint --insecure --sha1 https://google.com`

Let me know if any other changes are needed.